### PR TITLE
Feature/clubhouse

### PR DESCRIPTION
**Pull request by abdulkhaliqdev**
This project was created following The Odin Project specifications to create a social media where only members can see the authors of the posts. Rails was used to develop this project as Devise for the authentications, Simple Form for the forms, Bulma-rails for styling, and Better Errors for errors showcasing.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+  devise_for :users, :controllers => { registrations: 'registrations' }
+  root 'posts#index'
+  resources :posts
```
Comment (voscarmv) [inline config/routes.rb]: Please change this to 
```
resources :posts, only: [:new, :create, :index]
```

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    @post = Post.new
+  end
+
+  # GET /posts/1 or /posts/1.json
+  def show; end
+
+  # GET /posts/new
+  def new
+    @post = current_user.post.build
+  end
```
Comment (voscarmv) [inline app/controllers/posts_controller.rb]: Please use `@post = Post.new` for this action.

File: app/views/posts/_feed.html.erb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+          <% end %>
+          <%= link_to post, method: :delete, data: { confirm: "Are you sure you want to delete this tweeet?"} do %>
+            <span class="icon"><i class="fa fa-trash" aria-hidden="true"></i></span>
+          <% end %>
+        </div>
+      </nav>
+      <% end %>
+    </div>
+  <% end %>
+</div>
```
Comment (voscarmv) [inline app/views/posts/_feed.html.erb]: Instead of using `if` logic in your views, move it to helper functions.

**Review by voscarmv — CHANGES_REQUESTED** [review]
Impressive work. 💻 👀 ✨ 

- Your project works smoothly.
- The site is fully featured and it all works as it should.

## However, there are some changes required 👀 🧰 🔨 🔧 

- I only noticed a couple of details with your code. I've detailed them in the code comments. Please check them out and apply the changes requested there.

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by mke2111 — APPROVED** [review]
# **🎉 STATUS: APPROVED 🎉** 

## Hey @abdulkhaliqdev & @jadx2, 👋🏽 
 **Congratulations 🥳 🥂 . The clubhouse secrets work just fine and look user-friendly.**

I'm approving this PR but there is one mandatory change to make.

- [x] In your readme file, Kindly add `rails db:migrate` to your usage section before starting the server.

### Make the requested change and then proceed.
**You may now merge to the develop branch and submit a project completion form. 👍🏽** 

## Cheers!! 🍹 

## Reach me on slack for a chitchat 🆘  **Roy Mukuye**

**Pull request merged**